nixops/backends/container.py

```diff
@@ -63,9 +63,9 @@ def get_ssh_flags(self, *args, **kwargs):
         flags = super(ContainerState, self).get_ssh_flags(*args, **kwargs)
         flags += ["-i", self.get_ssh_private_key_file()]
         if self.host == "localhost":
-            flags.extend(MachineState.get_ssh_flags(self))
+            flags.extend(MachineState.get_ssh_flags(self, *args, **kwargs))
```

Comment:
so that [MachineState.get_ssh_flags](https://github.com/jraygauthier/nixops/blob/aa967926bfe5668a869fe0e714037c8de2e90548/nixops/backends/__init__.py#L233) gets its expected `scp` flag as specified by the caller. Otherwise it will default to `False`.

I honestly find these var args parameters confusing, I'm not sure why there was a need for it in the first place but I tried to make do with it.